govcd/edgegateway_test.go

```diff
@@ -465,3 +540,23 @@ func (vcd *TestVCD) Test_AddDNATRule(check *C) {
 
 	check.Assert(len(edge.EdgeGateway.Configuration.EdgeGatewayServiceConfiguration.NatService.NatRule), Equals, beforeChangeNatRulesNumber)
 }
+
+func TestGetPseudoUUID(t *testing.T) {
```

Comment:
This function should be in `edgegateway_unit_test.go` with tag `unit`, similar to `system_unit_test.go`

Reply:
fixed